tests/pr_test.go

```diff
@@ -143,3 +144,49 @@ func TestRunExistingResourcesExample(t *testing.T) {
 		logger.Log(t, "END: Destroy (existing resources)")
 	}
 }
+
+func TestRunStandardSolution(t *testing.T) {
+	t.Parallel()
+
+	options := testhelper.TestOptionsDefault(&testhelper.TestOptions{
+		Testing:       t,
+		TerraformDir:  standardSolutionTerraformDir,
+		Region:        "us-south",
+		Prefix:        "assistant-st-da",
+		ResourceGroup: resourceGroup,
+	})
+
+	options.TerraformVars = map[string]interface{}{
+		"plan":                "free",
+		"service_endpoints":   "public",
+		"resource_group_name": options.Prefix,
+	}
+
+	output, err := options.RunTestConsistency()
+	assert.Nil(t, err, "This should not have errored")
+	assert.NotNil(t, output, "Expected some output")
+}
+
+func TestRunStandardUpgradeSolution(t *testing.T) {
+	t.Parallel()
+
+	options := testhelper.TestOptionsDefault(&testhelper.TestOptions{
+		Testing:       t,
+		TerraformDir:  standardSolutionTerraformDir,
+		Region:        "us-south",
```

Comment:
Same as above. Any specific reason to run test always in us-south?

Reply:
Sure, I kept into consideration that the default region is `us-south` when tried provisioning the `watsonx Assistant` service. 
One other solution can be to assign randomly the available and supported regions but I also followed the other DA code like `postgres` and `Elastic`.

Do you suggest to use some other region instead?

Reply:
If no region restriction, then it should choose random.

Reply:
Done, same as [above](https://github.com/terraform-ibm-modules/terraform-ibm-watsonx-assistant/pull/11#discussion_r1893994317)